README.md

```diff
@@ -70,72 +93,30 @@ Working Group.
-odd-numbered versions (e.g. v5, v7, v9) are cut in October.
-
-In coordination with a new *odd-numbered* major release being cut, the
-previous *even-numbered* major version will transition to the Long Term Support
-plan. The transition to Long Term Support can happen either before or after
-the new major version is cut in a Semver-Minor release following
-[the process documented below](#marking-a-release-line-as-lts).
-
-Every major version covered by the LTS plan will be actively maintained for a
-period of 12 months from the date it enters LTS coverage. Following those 12
-months of active support, the major version will transition into "maintenance"
-mode for 18 additional months. Prior to Node.js 12 the active period was 18
-months and the maintenance period 12 months.
-
-The exact date that a release stream will be moved to LTS, moved between LTS
-modes, or deprecated will be chosen no later than the first day of the month.
-If it is to be changed, it will be done with no less than 14 days notice.
-
-Given this schedule, there will be no more than two active LTS releases at any
-given time, overlapping for a maximum period of six months.
-
-Once a major version enters LTS coverage, new features (semver-minor) may only
-be landed with consent of the Release working group. No semver-major
-changes other than those required for critical security fixes may be landed.
-
-Changes in an LTS-covered major version are limited to:
-
-1. Bug fixes.
-2. Security updates.
-3. Non-semver-major npm updates.
-4. Relevant documentation updates.
-5. Certain performance improvements where the risk of breaking existing
-   applications is minimal.
-6. Changes that introduce large amount of code churn where the risk of breaking
-   existing applications is low and where the change in question may
-   significantly ease the ability to backport future changes due to the
-   reduction in diff noise.
-
-Generally changes are expected to live in a *Current* release for at least 2
-weeks before being backported. It is possible for a commit to land earlier at
-the discretion of the Release working group and the maintainers of the LTS branches.
+New *semver-major* releases of Node.js are branched from `master` every six
+months. New even-numbered versions are released in April and odd-numbered
+versions in October.
 
-Once a release moves into Maintenance mode, only ***critical*** bugs,
-***critical*** security fixes, documentation updates, and updates to ensure
-consistency and usability of the N-API across LTS releases will be permitted.
-Unless a change is ***urgent*** it will be planned into a release once per
-quarter. Such releases will only be made when necessary.
+In coordination with a new *odd-numbered* major release, the previous
+*even-numbered* major version will transition to Long Term Support. The
+transition to Long Term Support will happen in a *semver-minor* release and can
+ happen either before or after the new major version is released.
```

Comment:
We may want to open a follow up issue to track this, if I recall this is the exact thing that caused the "two current releases" thing.

As we will also be transitioning Active LTS releases to maintenance at the same time now we may want to have slightly more robust / clear policy here.

To be clear, not blocking... this should be a follow up.

Reply:
Agreed - I think https://github.com/nodejs/Release/issues/495 had some of the conversation tied to it